nms/app/packages/magmalte/app/components/DataGrid.js

```diff
@@ -212,67 +213,74 @@ type Data = {
   statusCircle?: boolean,
   statusInactive?: boolean,
   status?: boolean,
+  tooltip?: string,
 };
 
 export type DataRows = Data[];
 
-type Props = {data: DataRows[], testID?: string};
+type Props = { data: DataRows[], testID?: string };
 
 export default function DataGrid(props: Props) {
   const classes = useStyles();
   const dataGrid = props.data.map((row, i) => (
     <Grid key={i} container direction="row">
-      {row.map((data, j) => (
-        <React.Fragment key={`data-${i}-${j}`}>
-          <Grid
-            item
-            container
-            alignItems="center"
-            xs={12}
-            md
-            key={`data-${i}-${j}`}
-            zeroMinWidth
-            className={classes.dataBlock}>
-            <Grid item xs={12}>
-              {data.collapse !== undefined && data.collapse !== false ? (
-                DataCollapse(data.category, data.value, data.collapse)
-              ) : (
-                <CardHeader
-                  data-testid={data.category}
-                  className={classes.dataBox}
-                  title={data.category}
-                  titleTypographyProps={{
-                    variant: 'caption',
-                    className: classes.dataLabel,
-                    title: data.category,
-                  }}
-                  subheaderTypographyProps={{
-                    variant: 'body1',
-                    className:
-                      data.obscure === true
-                        ? classes.dataObscuredValue
-                        : classes.dataValue,
-                    title: data.value + (data.unit ?? ''),
-                  }}
-                  subheader={
-                    data.statusCircle === true
-                      ? StatusIndicator(
-                          data.statusInactive || false,
-                          data.status || false,
-                          data.value + (data.unit ?? ''),
-                        )
-                      : data.icon
-                      ? DataIcon(data.icon, data.value + (data.unit ?? ''))
-                      : data.obscure === true
-                      ? DataObscure(data.value, data.category)
-                      : data.value + (data.unit ?? '')
-                  }
-                />
-              )}
+      {row.map((data, j) => {
+        const dataEntryValue = data.value + (data.unit ?? '');
+
+        return (
+          <React.Fragment key={`data-${i}-${j}`}>
+            <Grid
+              item
+              container
+              alignItems="center"
+              xs={12}
+              md
+              key={`data-${i}-${j}`}
+              zeroMinWidth
+              className={classes.dataBlock}>
+              <Grid item xs={12}>
+                {data.collapse !== undefined && data.collapse !== false ? (
+                  DataCollapse(data.category, data.value, data.collapse)
+                ) : (
+                    <Tooltip title={data.tooltip ?? dataEntryValue}>
```

Comment:
minor nit: We don't to add tooltip this way. CardHeader subheader "title" is the tooltip for the component.  You can set it as follows

subheaderTypographyProps={{
                          variant: 'body1',
                          className:
                            data.obscure === true
                              ? classes.dataObscuredValue
                              : classes.dataValue,
                         - title: dataEntryValue,
                         + title: props.tooltip ?? dataEntryValue,
                        }}